sdk/python/kfp/components/_structures.py

```diff
@@ -285,6 +322,12 @@ def verify_arg(arg):
                 elif isinstance(arg, OutputPathPlaceholder):
                     if arg.output_name not in self._outputs_dict:
                         raise TypeError('Argument "{}" references non-existing output.'.format(arg))
+                elif isinstance(arg, InputUriPlaceholder):
+                    if arg.input_name not in self._inputs_dict:
+                        raise TypeError('Argument "{}" references non-existing input.'.format(arg))
+                elif isinstance(arg, OutputUriPlaceholder):
```

Comment:
Can we merge this with line 322? Together with the branch for `OutputPathPlaceholder`?

Similar for `InputUriPlaceholder`

Reply:
Done.